src/util/redirects-raw.txt

```diff
@@ -10,7 +10,8 @@
 /docs/writing-docs/doc-block-story               /docs/api/doc-block-story                                                      301
 /docs/writing-docs/doc-block-typeset             /docs/api/doc-block-typeset                                                    301
 /docs/workflows/testing-with-storybook/          /docs/writing-tests/introduction/                                              301
-/docs/workflows/unit-testing/                    /docs/writing-tests/importing-stories-in-tests/                                301
```

Comment:
@jonniebigodes — Your original change was:

```diff
- /docs/workflows/unit-testing/  /docs/writing-tests/importing-stories-in-tests/                                301
+ /docs/workflows/unit-testing/  /docs/writing-tests/unit-testing/                                                        301
```

Which would result in the current URL, `/docs/writing-tests/importing-stories-in-tests/` 404'ing.

Instead, we need to have redirects for both the old URL, `/docs/workflows/unit-testing/` and the current one, which both point to the new one.